nova/virt/vmwareapi/driver.py

```diff
@@ -445,60 +428,59 @@ def get_available_nodes(self, refresh=False):
 
         This driver supports only one compute node.
         """
+        # As this is called before polling each node
+        self._vc_state.get_host_stats(True)
```

Comment:
I don't get the comment. "as this is called before polling each node, we ..."? What I get is, that we need this to be called before polling each node, but I don't really get, why. Can we add that (in short)? And maybe da a `refresh=True` again, so it's clear what we `True` here?

Reply:
Sure, I expand that comment a bit